pom.xml

```diff
@@ -210,6 +229,87 @@
 						</includes>
 					</configuration>
 				</plugin>
+				<plugin>
+					<groupId>org.apache.maven.plugins</groupId>
+					<artifactId>maven-jar-plugin</artifactId>
+					<version>${maven-jar-plugin.version}</version>
+				</plugin>
+				<plugin>
+					<groupId>org.apache.maven.plugins</groupId>
+					<artifactId>maven-javadoc-plugin</artifactId>
+					<version>${maven-javadoc-plugin.version}</version>
+				</plugin>
+				<plugin>
+					<groupId>org.apache.maven.plugins</groupId>
+					<artifactId>maven-checkstyle-plugin</artifactId>
+					<version>${maven-checkstyle-plugin.version}</version>
+					<dependencies>
+						<dependency>
+							<groupId>com.puppycrawl.tools</groupId>
+							<artifactId>checkstyle</artifactId>
+							<version>${puppycrawl-tools-checkstyle.version}</version>
+						</dependency>
+						<dependency>
+							<groupId>io.spring.javaformat</groupId>
+							<artifactId>spring-javaformat-checkstyle</artifactId>
+							<version>${spring-javaformat-checkstyle.version}</version>
+						</dependency>
+						<dependency>
+							<groupId>org.springframework.cloud</groupId>
+							<artifactId>spring-cloud-build-tools</artifactId>
+							<version>${spring-cloud-build.version}</version>
+						</dependency>
+						<dependency>
+							<groupId>io.spring.nohttp</groupId>
+							<artifactId>nohttp-checkstyle</artifactId>
+							<version>${nohttp-checkstyle.version}</version>
+						</dependency>
+					</dependencies>
+					<executions>
+						<execution>
+							<id>checkstyle-validation</id>
+							<phase>validate</phase>
+							<inherited>true</inherited>
+							<configuration>
+								<skip>${disable.checks}</skip>
+								<configLocation>checkstyle.xml</configLocation>
+								<headerLocation>checkstyle-header.txt</headerLocation>
+								<propertyExpansion>
+									checkstyle.build.directory=${project.build.directory}
+									checkstyle.suppressions.file=${checkstyle.suppressions.file}
+									checkstyle.additional.suppressions.file=${checkstyle.additional.suppressions.file}
+								</propertyExpansion>
+								<consoleOutput>true</consoleOutput>
+								<includeTestSourceDirectory>
+									${maven-checkstyle-plugin.includeTestSourceDirectory}
+								</includeTestSourceDirectory>
+								<failsOnError>${maven-checkstyle-plugin.failsOnError}
+								</failsOnError>
+								<failOnViolation>
+									${maven-checkstyle-plugin.failOnViolation}
+								</failOnViolation>
+							</configuration>
+							<goals>
+								<goal>check</goal>
+							</goals>
+						</execution>
+						<execution>
+							<id>no-http-checkstyle-validation</id>
```

Comment:
Do you think we should remove this check? I don't think we used this one before despite it being inherited.

Reply:
Removed all `nohttp` stuff.